game_functions/task_function.py

```diff
@@ -7,7 +7,7 @@ class task_functions:
     def __init__(self, win):
         self.trialClock=core.Clock()
         image_path = 'Images/Tasks/'
-        self.fixation=visual.TextStim(win, ori=0, font=u'Arial', pos=[0, 0], color=u'white',text=u'+')
+        self.fixation=visual.TextStim(win, ori=0, font=u'Arial', pos=[0, 0], height=50, color=u'white',text=u'+')
```

Comment:
because the merge wasn't immediately implemented/merged. By the time of testing, I was already working on other things. When it's pushed to github, it's automatically included in the merge.